base/test/SConscript

```diff
@@ -113,8 +113,33 @@ test_suite = [
     conn_info_test,
     ]
 
+#Required by base:test
+#controller/src/bgp
```

Comment:
This is a bit brute force, pls make a note for follow, we will make this a bit more elegant.